homeassistant/setup.py

```diff
@@ -626,27 +633,139 @@ def async_get_loaded_integrations(hass: core.HomeAssistant) -> set[str]:
     return integrations
 
 
+class SetupPhases(StrEnum):
```

Comment:
In the future PR I'm gonna make this available in diagnostics so we can get significantly more information on timings when something is going wrong